tests/lsdb/catalog/test_validators.py

```diff
@@ -0,0 +1,60 @@
+import numpy as np
+import numpy.testing as npt
+import pytest
+
+from lsdb.core.search.polygon_search import get_cartesian_polygon
+from lsdb.core.search.validators import CoordinatesValidator
+
+
+def test_wrap_right_ascension():
+    # Does not need to be wrapped
+    ra = [20, 150, 350]
```

Comment:
Can you add some test cases that use floats (e.g. `23.4`, `-89.999`)? Modular division and floats make me nervous.

Reply:
This still makes me nervous.